Tests/Unit/NetworkServer/IotHubDeviceInfoConversionTests.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable enable
+
+namespace LoRaWan.Tests.Unit.NetworkServer
+{
+    using global::LoraKeysManagerFacade;
+    using Newtonsoft.Json;
+    using Xunit;
+
+    public sealed class IotHubDeviceInfoConversionTests
+    {
+        [Fact]
+        public void Serialization_And_Deserialization_Preserves_Information_Using_JsonNet()
```

Comment:
Since it gets deserialized into a _different_ type, I would recommend to reflect that in the test fact:

```suggestion
        public void Can_Be_Json_Deserialized_As_IoTHubDeviceInfo_From_NetworkServer()
```